apps/src/craft/designer/craft.js

```diff
@@ -335,6 +347,7 @@ Craft.init = function (config) {
         mc.add(new Hammer.Press({time: 150}) );
         mc.add(new Hammer.Tap() );
         mc.on("pan", onDrag);
+        mc.on('panend pancancel', onDragEnd);
```

Comment:
Are these events documented somewhere in the Hammer.js docs?

Reply:
[yep!](http://hammerjs.github.io/recognizer-pan/)